Composer/plugins/azurePublish/src/node/azureResourceManager/azureResourceManager.ts

```diff
@@ -203,6 +205,214 @@ export class AzureResourceMananger {
     }
   }
 
+  /**
+   * QnA Resource depends on serveral components, including appinights and web config
+   * @param config 
+   */
+  public async deployQnAReource(config: QnAResourceConfig): Promise<{ endpoint: string; subscriptionKey: string }> {
+    try {
+      this.logger({
+        status: BotProjectDeployLoggerType.PROVISION_INFO,
+        message: 'Deploying QnA Resource ...',
+      });
+
+      // initialize the name
+      const qnaMakerSearchName = `${config.accountName}-search`.toLowerCase().replace('_', '');
```

Comment:
in all of the other methods, this is `config.name` -- let's keep it the same instead of introducing a different key name in this one funciton.

Reply:
> in all of the other methods, this is `config.name` -- let's keep it the same instead of introducing a different key name in this one funciton.

Fixed.